rtmlib/tools/solution/pose_tracker.py

```diff
@@ -178,6 +178,10 @@ def __call__(self, image: np.ndarray):
 
             self.track_ids_last_frame = track_ids_current_frame
 
+            # reorder keypoints, scores according to track_id
+            keypoints = np.array([keypoints[i] for i in self.track_ids_last_frame])
+            scores = np.array([scores[i] for i in self.track_ids_last_frame])
+
```

Comment:
```suggestion
            # reorder keypoints, scores according to track_id
            keypoints = keypoints[self.track_ids_last_frame]
            scores = scores[self.track_ids_last_frame]

```
Hi @davidpagnon , thanks for your contribution. Could you give an example to demonstrate the necessity of reordering? BTW, I think numpy array can simply be reordered via the index so that we can avoid the for loop.